### DISCUSSION TITLE
Power usage

### DISCUSSION THREAD

**Discussion by pprn97** [lllyasviel/stable-diffusion-webui-forge#733] [Q&A]
I am using 4060Ti (160 W)
During rendering image, my gpu always runs at max watt ~160W
Is there any way to limit power usage 
I am using SD v1.5 WebUi

**Comment by thePlott** [comment]
The only way I can think of is to under-clock (as opposed to over-clocking) your gpu. This may come at some cost ie generation times may increase.
